src/api/restaurants.js

```diff
@@ -17,7 +17,7 @@ export const fetchRestaurantById = async (id) => {
     const { data } = await axios.get(`${URL}/restaurants/${id}`);
     return data;
   } catch (error) {
-    console.log(error)
+    console.log(error);
```

Comment:
I suggest an errorHandler utility for importing as we discussed before:
```ts
function errorHandler(exception: unknown): string {
    const message = (typeof exception === 'string') ? exception.toUpperCase()
        : (exception instanceof Error) ? exception.message : 'Something was wrong';

    return message;
}

export default errorHandler;
```
Into a try-catch:
```ts
} catch (error: unknown) {
      return res.status(422).json({
        "error": [errorHandler(error), []].join(' ')
      });
}
```